cypari2/pari_instance.pyx

```diff
@@ -385,22 +387,22 @@ def prec_words_to_dec(long prec_in_words):
 
 # Callbacks from PARI to print stuff using sys.stdout.write() instead
 # of C library functions like puts().
-cdef PariOUT sage_pariOut
+cdef PariOUT python_pariOut
 
-cdef void sage_putchar(char c):
+cdef void python_putchar(char c):
     cdef char s[2]
     s[0] = c
     s[1] = 0
-    sys.stdout.write(s)
+    sys.stdout.write(to_string(<bytes> s))
```

Comment:
Does not work as is since sys.stdout is sometimes redirected (e.g. doctest). I used

    try:
        sys.stdout.buffer.write(s)
    except AttributeError:
        sys.stdout.write(to_string(s))

See b4410fc